### ISSUE TITLE
oneOf issue

### ISSUE THREAD

**Issue by sgurin** [josdejong/jsoneditor#1548]
Hi.

Take a look on the example https://sgurin.github.io/jsoneditor/examples/26_autocomplete_by_schema.html

When I am trying to set profession -> level there is only one option available 

![image](https://github.com/josdejong/jsoneditor/assets/25930200/e3f0041b-6e26-4811-9bed-3565e9116fe6)

but schema contains 3 refs

      "profession": {
        "oneOf": [
          {
            "$ref": "junior"
          },
          {
            "$ref": "experienced"
          },
          {
            "$ref": "senior"
          },
        ]
      },

Looks like refs don't work

**Comment by josdejong** [comment]
Yes you are right, thanks for bringing this up. So to fix this, the code that creates the enum should proabably merge multiple enums in a case like `oneOf`. And maybe when there is a conflict and one of the options of `oneOf` does not contain an enum for that field, it should not create an enum.

Anyone interested in improving on this?
